roles/lmod/templates/lmod-ubuntu.j2

```diff
@@ -15,10 +12,10 @@ if [ $(id -u) -ne 0 ] ; then
     export LMOD_FULL_SETTARG_SUPPORT=no
     export LMOD_COLORIZE=yes
     export LMOD_PREPEND_BLOCK=normal
-    export MODULEPATH_ROOT="{{ sm_module_root }}"
-    export MODULEPATH="{{ sm_module_path }}"
+    export MODULEPATH_ROOT="/sw/modules"
```

Comment:
Is there a reason we replaced this variable with a hardcoded path?

Reply:
good catch, it was a leftover. Fixed